db/migrate/20181025230931_add_comments_to_vms_table.rb

```diff
@@ -0,0 +1,81 @@
+    change_column_comment :vms, :busy, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :boot_time, "The last time the VM was started."
+    change_column_comment :vms, :cloud, "Indicates whether or not the VM/Template is cloud or infra."
+    change_column_comment :vms, :cloud_network_id, "ID of the cloud (virtual) network the VM/Template is associated with."
+    change_column_comment :vms, :cloud_subnet_id, "Subnet ID associated with the VM."
+    change_column_comment :vms, :cloud_tenant_id, "ID of the cloud tenant associated with the VM."
+    change_column_comment :vms, :config_xml, "The VM or Template configuration in XML format. Deprecated."
+    change_column_comment :vms, :connection_state, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :cpu_affinity, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :cpu_hot_add_enabled, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :cpu_hot_remove_enabled, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :cpu_limit, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :cpu_reserve, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :cpu_reserve_expand, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :cpu_shares, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :cpu_shares_level, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :created_on, "The date the VM was added to the app inventory."
+    change_column_comment :vms, :deprecated, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :description, "A custom description that users can add within the app."
+    change_column_comment :vms, :ems_created_on, "Date the associated EMS was created."
+    change_column_comment :vms, :ems_id, "ID of the ExtManagementSystem associated with the VM."
+    change_column_comment :vms, :evm_owner_id, "ID of VM owner from product database."
+    change_column_comment :vms, :ems_ref_obj, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :ems_cluster_id, "ID of associated cluster."
+    change_column_comment :vms, :fault_tolerance, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :flavor_id, "ID of associated flavor."
+    change_column_comment :vms, :format, "The format of the VM's disk, such as VMDK."
+    change_column_comment :vms, :guid, "A value used to uniquely identify the VM internally."
+    change_column_comment :vms, :id, "The internal database ID."
+    change_column_comment :vms, :host_id, "ID of the host that the VM/Template resides on."
+    change_column_comment :vms, :last_perf_capture_on, "Timestamp for last performance metrics capture."
+    change_column_comment :vms, :last_scan_on, "Timestamp for last SmartState Analysis."
+    change_column_comment :vms, :last_scan_attempt_on, "Timestamp for last attempt to perform SmartState Analysis."
+    change_column_comment :vms, :last_sync_on, "[DEFINITION NEEDED]"
+    change_column_comment :vms, :linked_clone, "[DEFINITION NEEDED]"
```

Comment:
A boolean indicating if a VM is a "linked clone" aka its base disk is a differencing disk pointing back to a snapshot of another VM.  This is a way of creating a high number of clones without using much storage space.